src/ApiService/ApiService/OneFuzzTypes/Model.cs

```diff
@@ -87,6 +94,26 @@ bool DebugKeepNode
 ) : EntityBase();
 
 
+public partial record ProxyForward
+(
+	string Region,
+	int SrcPort,
+	int DstPort,
+	int DstIp
+) : EntityBase();
```

Comment:
you don't need to derive form EntityBase if this is not meant to be stored in the db

Reply:
Proxy Forward is currently stored in the Azure tables.